src/Tools/Consolonia.PreviewHost/App.cs

```diff
@@ -73,6 +74,8 @@ public override void OnFrameworkInitializationCompleted()
                     };
 
                 base.OnFrameworkInitializationCompleted();
+
+                this.InitializeConsolonia();
```

Comment:
We could call it from base class, but now spreading to all inheritors, why is that?